tests/create_all_guardduty_standard_resources/main.tf

```diff
@@ -136,6 +137,23 @@ module "guardduty_standard_resources" {
       }
     }
   }
+
+  detector_features = [
+    {
+      name   = "LAMBDA_NETWORK_LOGS"
+      status = "ENABLED"
+      region = data.aws_region.current.name
+    },
+    {
+      name   = "EKS_RUNTIME_MONITORING"
+      status = "ENABLED"
+      region = data.aws_region.current.name
+      additional_configuration = {
+        name   = "EKS_ADDON_MANAGEMENT"
+        status = "ENABLED"
+      }
+    },
+  ]
```

Comment:
Something is wrong with this branch. I see this in the PR:

```hcl
  detector_features = [
    {
      name   = "LAMBDA_NETWORK_LOGS"
      status = "ENABLED"
      region = data.aws_region.current.name
    },
    {
      name   = "EKS_RUNTIME_MONITORING"
      status = "ENABLED"
      region = data.aws_region.current.name
      additional_configuration = {
        name   = "EKS_ADDON_MANAGEMENT"
        status = "ENABLED"
      }
    },
  ]
```

But see this when I check the branch directly:

```hcl
  detector_features = [
    {
      name   = "EKS_RUNTIME_MONITORING"
      status = "ENABLED"
      region = data.aws_region.current.name
      additional_configuration = {
        name   = "EKS_ADDON_MANAGEMENT"
        status = "ENABLED"
      }
    }
  ]
```